ee/clickhouse/views/person.py

```diff
@@ -37,6 +38,21 @@ def should_paginate(results, limit: Union[str, int]) -> bool:
     return len(results) > int(limit) - 1
 
 
+def get_funnel_actor_class(filter: Filter) -> Callable:
+    funnel_actor_class: Type[ClickhouseFunnelBase]
+    if filter.funnel_viz_type == FunnelVizType.TRENDS:
```

Comment:
time to convert?

Reply:
We don't return actors for time to convert so it's not needed in this function